keps/prod-readiness/sig-scheduling/5142.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 5142
+alpha:
+  approver: "@wojtek-t"
```

Comment:
/cc @wojtek-t

Reply:
ack - waiting for sig buy-in (the PRR is looking good enough, I'm just not sure I understand the change itself - see my comment below).